### PR TITLE
Google cal redux

### PR DESCRIPTION
**Pull request by reginafcompton**
@hancush - could you take a look, particularly, the changes in utils.py?

### CODE REVIEW

File: nyc/utils.py [hunk 1]
```diff
@@ -1,68 +1,23 @@
...
-    credentials = get_credentials()
-    http = credentials.authorize(httplib2.Http())
-    service = discovery.build('calendar', 'v3', http=http)
+def create_google_cal_link(event):
+    event_url = ('https://www.google.com/calendar/render?action=TEMPLATE&text={text}'
+        '&dates={start_time}/{end_time}'
+        '&details={details}'
+        '&location={location}'
+        '&sf=true&output=xml').format(text=event.name.strip(),
+                start_time=event.start_time.strftime("%Y%m%dT%H%M%SZ"),
```
Comment (hancush) [inline nyc/utils.py]: Perhaps break the time format into a var to be used in both `strftime` operations, like:

```python
time_fmt = '%Y%m%dT%H%M%SZ'
# then
event.start_time.strftime(time_fmt)
event.start_time + timedelta(hours=2).strftime(time_fmt)
```

File: nyc/utils.py [hunk 2]
```diff
@@ -1,68 +1,23 @@
...
-      'start': {
-        'dateTime': event.start_time.isoformat(),
-        'timeZone': settings.TIME_ZONE,
-      },
-      'end': {
-        'dateTime': (event.start_time + timedelta(hours=2)).isoformat(),
-        'timeZone': settings.TIME_ZONE,
-      },
-    }
+    return event_url.strip().replace(' ', '+')
```
Comment (hancush) [inline nyc/utils.py]: Doesn't look like you'll ever have leading or trailing whitespace in the generated URL, so I think we can lose the `strip()` call here.

File: tests/test_calendar.py [hunk 3]
```diff
@@ -22,18 +22,4 @@ def test_ical_export(django_db_setup, client):
...
-    url = reverse('nyc:google_calendar_export', kwargs={'slug': event.slug})
-    response = client.get(url)
-
-    event_arg, _ = mock_gcal_helper.call_args
-    # The event called in google_calendar_export_helper should be the same as the event passed to the view.
-    assert event_arg[0] == event
-    # The response should redirect.
-    assert response.status_code == 302
\ No newline at end of file
+    assert response['Content-Disposition'] == 'attachment; filename={}.ics'.format(event.slug)
```
Comment (hancush) [inline tests/test_calendar.py]: RIP `mock_gcal_helper`! Worth it to test your Gcal URL generator method? Could be as simple as:

```python
e = Event.objects.get(pk=1)
url = create_google_cal_link(e)
assert url == HARD_CODED_URL_WE_EXPECT
```

File: nyc/utils.py (hunk 1 above)
Comment (reginafcompton) [reply nyc/utils.py]: YES

**Pull request merged**